CodingClubSource/CodingClubTests/StringLibraryTests.cpp

```diff
@@ -24,7 +24,7 @@ namespace CodingClubTests
 		TEST_METHOD(String_RemoveVowels_Returns_InputString_With_Vowels_Removed)
 		{
 			// Given
-			std::string input = "Cornwall";
+			std::string input = "CornwAllO";
```

Comment:
This is fine, but would probably want a separate unit test for upper and maybe another for mixed case. Helps narrow down the problem if one of the cases stops working.